src/Microsoft.Azure.SignalR.Protocols/ServiceMessage.cs

```diff
@@ -85,6 +85,14 @@ public interface IAckableMessage
         int AckId { get; set; }
     }
 
+    /// <summary>
+    /// Interface of connection ack-able message 
+    /// </summary>
+    public interface IConnectionAckableMessage : IAckableMessage
```

Comment:
what about `IAckableMessageForConnection`?

Reply:
Shall we avoid preposition word in middle of naming and it seems not align with most others? For example, we don't use ConnectionMessageForOpen nor JoinGroupMessageForUser ?